src/utils/Dom.ts

```diff
@@ -19,6 +19,11 @@ export interface IOffset {
 export class Dom {
   private static CLASS_NAME_REGEX = /-?[_a-zA-Z]+[_a-zA-Z0-9-]*/g;
   private static ONLY_WHITE_SPACE_REGEX = /^\s*$/;
+  /**
+   * Use javascript events intead of Jquery events.
+   * @type boolean
+   */
+  public static useNativeJavascriptEvents = false;
```

Comment:
Suggestion:

Rename property to `useNativeJavaScriptEvents` (Javascript -> JavaScript)

Documentation:

Whether to always register, remove, and trigger events using standard JavaScript rather than attempting to use jQuery first.